kglib/kgcn/pipeline/pipeline.py

```diff
@@ -136,3 +124,57 @@ def make_blank_embedder():
 
     _, _, _, _, _, solveds_tr, solveds_ge = tr_info
     return ge_graphs, solveds_tr, solveds_ge
+
+
+def configure_embedders(node_types, attr_embedding_dim, categorical_attributes, continuous_attributes):
```

Comment:
this is a bit awkward to read... maybe it can be split into separate file/class?

Reply:
Agreed, it's generally awkward and needs more architectural work, including how to expose this configurability to the user. Intending to address this in a separate PR as it's a big task

Reply:
Added #103